integration/kubernetes/helm-chart/alluxio/templates/worker/daemonset.yaml

```diff
@@ -88,6 +88,18 @@ spec:
       {{- if .Values.imagePullSecrets }}
 {{ include "alluxio.imagePullSecrets" . | indent 6 }}
       {{- end}}
+      {{- if and (eq .Values.shortCircuit.enabled true) (eq .Values.shortCircuit.volumeType "hostPath") }}
```

Comment:
One alternative is, instead of initContainer and root perm, would it be better if you need the hostPath to exist (`Directory` instead of `DirectoryOrCreate`). Would a `file not found` be more straightforward to the user than `perm denied`?